bot.js

```diff
@@ -4,10 +4,17 @@ module.exports = class Bot {
   constructor(client, botApiKey) {
     this.botApiKey = botApiKey;
     this.client = client;
+
+    this.lastMessage = {author:{bot:false}};
+
+    this.retrieveLastMessage = () => {
+      return this.lastMessage;
+    }
   }
   
   init() {
     this.client.on('message', (message) => {
+      this.lastMessage = message;
```

Comment:
Unsure of how to test this locally... would it perhaps be better to declare this variable via `let` outside of the class declaration? Not too familiar with the mutability of `this`-bound variables.